go.mod

```diff
@@ -48,6 +47,7 @@ require (
 	github.com/microsoft/go-mssqldb v1.8.0
 	github.com/neilotoole/slogt v1.1.0
 	github.com/nucleuscloud/go-antlrv4-parser v0.0.0-20240830015744-041b46c70aa5
+	github.com/nucleuscloud/sql-adapter v0.0.0-20250327235651-6f5a05a7cedc
```

Comment:
This ultimately is the fix as I updated the insert logic to just on conflict do nothing instead of conflict.